source/buildings/university.md

```diff
@@ -55,27 +55,293 @@ Page 2 of the GUI will show you the options for each research tree.
+
+### Sixth Column
+
+You can only choose one of these!
+
+| Name | Prerequisites | Requirements | Effects |
+| ---- | ------------- | ------------ | ------- |
+| Whirlwind | Mightly Cleave | 32 blocks of iron | Increases Knights' damage by 2 hearts (cumulative) |
+| Deadly Aim | Wounding Shot | 256 flint | Increases Archers' damage by 2 hearts (cumulative) |
+| Captain of the Guard | Captain Training | 27 shields | Increases the chance for Knights to block attacks by 50% |
+| Master Bowman | Rapid Shot | 27 bows | 50% chance for Archers to shoot 2 arrows at the same time |
+| Master Swordsman | Provost | 64 diamonds | Increases Knights' armor by 100% |
+| Agile Archer | Improved Evasion | 64 diamonds | Increases Archers' armor by 100% |
+| Diamond Skin | Steel Armor | 64 diamonds | Increases Guards' armor's durability by 100% |
+| Full Retreat | Retreat | 64 emeralds | When Guards are fleeing, the damage they take is reduced by 100% |
+| Hotfoot | Flee | 64 emeralds | Increases the fleeing speed of Guards by 50% |
+
+## Civilian
 <br>
-Civilian
 <img src="../../assets/images/gui/universitycivilian.png" class="img-fluid mx-auto" alt="University Civilian GUI Page">
+<br>
+
+### First Column
+
+| Name | Requirements | Effects |
+| ---- | ------------ | ------- |
+| Stamina | 1 carrot | Unlocks the [Hospital](../../source/buildings/hospital) |
+| Keen | [House](../../source/buildings/house) at least level 3; 3 books | Unlocks the [Library](../../source/buildings/library) |
+| First Aid | [Town Hall](../../source/buildings/townhall) at least level 1; 8 hay bales | Increases colonists' health by 1 heart |
+| Higher Learning | House at least level 3; 3 books | Unlocks the [School](../../source/buildings/school) |
+
+### Second Column
+
+| Name | Prerequisites | Requirements | Effects |
+| ---- | ------------- | ------------ | ------- |
+| Band Aid | Stamina | Library at least level 2; 1 golden carrot | Every tick that a colonist is healing, they heal by 10% more |
+| Resistance | Stamina | [Restaurant](../../source/buildings/restaurant) at least level 2; 1 golden apple | Colonists can heal with only 95% saturation |
+| Outpost | Keen | House at least level 4; 64 steak | Lets you have over 25 colonists |
+| Diligent | Keen | Library at least level 2; 6 books | Increases the rate colonists level up by 5% |
+| Rails | Keen | [Courier's Hut](../../source/buildings/courier) at least level 3; 64 rails | Lets colonists use rails to get around. You don't need to provide them with minecarts, as they automagically create their own for each trip |
+| First Aid II | First Aid | Town Hall at least level 2; 16 hay bales | Increases colonists' health by 2 hearts |
+| Circus | First Aid | Restaurant at least level 2; 1 cake | Increases colonists' happiness by 5% |
+| Gourmand | First Aid | Restaurant at least level 2; 32 cookies | Colonists' saturation is increased by 10% |
```

Comment:
Oops, to clarify, this is still per food, but food increases the colonists' saturation, and doesn't have saturation itself per se.
```suggestion
| Gourmand | First Aid | Restaurant at least level 2; 32 cookies | Food consumed by colonists' is 10% more effective |
```

Reply:
Maybe "Food consumed by colonists increases their saturation by 10%"?

Reply:
It does not increase the saturation of the colonists by ten percent. It increases the amount that the food restores by ten percent. So the buff is according to the food, not the colonist. So that may be more, or less, than 10% of the total saturation of the colonist. That is why that suggestion makes the most sense.

Reply:
In my opinion the suggestion could also refer to increasing the food's hunger by 10%. What about "Food consumed by colonists is 10% more effective to increase their saturation"?

Reply:
I see very little difference in the interpretation of those two statements, but if you feel the clarification necessary then sure. 
"Food is 10% more effective at restoring colonists' saturation"
I guess we could be here til Christmas trying to refine this, but you might just have to pick one :smiley: